content/kubermatic/master/advanced/user_settings/_index.en.md

```diff
@@ -0,0 +1,27 @@
++++
+title = "Kubermatic User Settings"
+date = 2020-04-01T14:07:15+02:00
+weight = 1
+pre = "<b></b>"
++++
+
+This manual explains how to manage Kubermatic user settings via the UI.
+
+### Accessing the User Settings
+To access the user settings click the `User Settings` entry in the user menu:
+
+![User menu](/img/advanced/user_settings/menu.png)
+
+### User Settings Overview
+
+![User Settings view](/img/advanced/user_settings/view.png)
+
+The user settings view is devided into two columns: 
+The left side contains all account details, such as the name and the email of the user, whereas the right side lists all possible settings that a user can make. 
+
+* `Items per page` will change the number of visible items per page in a table. It will be applied to all tables within kubermatic.
+* `Theme` lists all available themes for the dashboard so the user can select its prefered one. 
```

Comment:
I think we could add a reference to the theme documentation (as soon as it exists).
Btw: Theming as a new topic or within https://docs.loodse.com/kubermatic/master/advanced/custom_ui/ ? WDYT?

Reply:
We also have the theming docs in dashboard's repo, it would be good if someone would move it here. We can use the same topic if you want.